app/assets/javascripts/account/reports.js

```diff
@@ -0,0 +1,83 @@
+document.addEventListener('turbolinks:load', () => {
+  var graphs = document.getElementById('graphs');
+
+  if(graphs) {
+    var ids = graphs.querySelectorAll('*[id]');
+
+    for(var i = 0; i < ids.length; i++) {
+      chart(JSON.parse(ids[i].dataset.graph), ids[i]);
+    }
+  }
+});
+
+function chart(workspaceReport, element) {
```

Comment:
rename "chart" function to something better
its too general
I told before this